index.html

```diff
@@ -519,10 +528,6 @@ <h3>
                   </li>
                 </ol>
               </li>
-              <li>If |signal| is not `null` and |signal|’s <a>aborted flag</a>
```

Comment:
Since obtaining permission is asynchronous then the release a wake lock algorithm could have run before we run the acquire a wake lock algorithm below. Does it make sense to check that `promise` is still pending?

Reply:
I understand the issue you've mentioned, but it's not clear to me where you're suggesting to check `promise`'s status.

In parallel (pun unintended), coming to think of it, one can call `AbortController.abort()` at any point during those parallel steps; maybe that's why @kenchris had so many checks for the aborted flag in the steps, but I feel we could avoid all the repeated checks somehow.

Reply:
Yes, I had a check before each async operation. I didn't see the best way to avoid this or make the spec text nicer